### PR TITLE
Add script to make it easier to publish messages to Google Pub/Sub

### PR DESCRIPTION
**Pull request by NickGrantONS**
# Motivation and Context
Currently, there's no easy way to publish messages to the Pub/Sub emulator on our local machines.

# What has changed
Added a script that makes it easy to publish to the Pub/Sub emulator (or indeed anywhere).

# How to test?
Update your pipenv by running `pipenv install --dev` then try publishing a message...

Run `./publish_message.sh event_receipt` and then type in a JSON message. Press CTRL-D when you're done.

# Links
Trello: https://trello.com/c/V4q1Fp1R

### CODE REVIEW

**Comment by NickGrantONS** [comment]
[Figure out how to cURL a message onto a pubsub emulator topic, mainly for Yogi but might be handy for us too (5)](https://trello.com/c/V4q1Fp1R/2644-figure-out-how-to-curl-a-message-onto-a-pubsub-emulator-topic-mainly-for-yogi-but-might-be-handy-for-us-too-5)

File: publish_message.sh [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+    echo "Usage: publish_message.sh <TOPIC>"
+    exit -1
+fi
+
+source .env
+
+echo "Pipe in message to send to topic '$1', or paste and it ctrl-d when done"
+message=$(cat)
+
+PUBSUB_EMULATOR_HOST=$PUBSUB_SETUP_HOST python publish_message.py $1 $message
```
Comment (ryangrundy7) [inline publish_message.sh:14]: This should be `pipenv run python` instead of just `python`

Comment (NickGrantONS) [reply publish_message.sh]: changed it

**Review by ryangrundy7 — APPROVED** [review]
Able to push multiline json messages

**Pull request merged**